fdbserver/workloads/MetaclusterManagementWorkload.actor.cpp

```diff
@@ -412,7 +426,15 @@ struct MetaclusterManagementWorkload : TestWorkload {
 					if (e.code() == error_code_tenant_already_exists && retried && !exists) {
 						Optional<TenantMapEntry> entry = wait(MetaclusterAPI::tryGetTenant(self->managementDb, tenant));
 						ASSERT(entry.present());
+						retrySuccess = true;
```

Comment:
This might be slightly clearer if you did something like `createEntry.assignedCluster = entry.get().assignedCluster` or `createEntry = entry.get()`, and then remove the `retrySuccess` variable from the assert below, though I'll leave that up to you.